GoogleSheetI18n/Integrations/Web/I18nController.cs

```diff
@@ -86,5 +101,51 @@ public object Get()
                 _i18nOptions.SubscriptionUrl
             };
         }
+
+        private object GetWarnings(IList<I18nSheet> i18nSheets)
+        {
+            List<string> warnings = new List<string>();
+            foreach (var sheet in i18nSheets)
+            {
+                if (sheet != null)
+                {
+                    var possibleKeys = sheet.ValueRange.Values.Skip(1).Select(x => x.FirstOrDefault()).ToList();
+
+                    var text = _i18nGoogleClient.GetExtractedTranslations(sheet.SheetName.FromPascalCaseToDashLowerCase(), possibleKeys);
+
+                    if (!string.IsNullOrEmpty(text))
+                    {
+                        warnings.Add(text);
+                    }
+
+
+                    var languages = sheet.GetSupportedLanguages();
+                    warnings.AddRange(languages.Warnings);
+
+                    foreach (var lng in languages)
+                    {
+                        warnings.AddRange(sheet.GetTranslations(lng).Warnings);
+
+                    }
+                }
+            }
+
+            return new
+            {
+                Message = CreateErrorMessage(warnings)
+            };
+        }
+
+        private string CreateErrorMessage(List<string> warnings)
```

Comment:
Looks like, this method shouldn't be here.

Reply:
removed